app/routes/bdb/utils.tsx

```diff
@@ -158,6 +158,18 @@ export const getSemesterSlugById = (
   return `${semester.year}${semester.semester === 'autumn' ? 'h' : 'v'}`;
 };
 
+export const getSemesterSlugOffset = (
```

Comment:
Is this not kind of over engineered? Do we ever need an offset greater than 1? I feel like it would be more practical to just get a direction as an arg; `previous`/`next` or `left`/`right`

Reply:
Yeah you are probably right, will change it 👍